Assets/Scripts/Actors/FootstepSound.cs

```diff
@@ -1,9 +1,13 @@
 ﻿using UnityEngine;
 using System.Collections;
 
-public class FootstepSound : MonoBehaviour {
+public class FootstepSound : MonoBehaviour 
+{
+	[SerializeField] AudioSource[] footstepSounds;
 
-	[SerializeField] void PlayerWalkSound(){
-		SoundManager.instance.PlaySoundAtPosition ("Footstep", transform.position-transform.up);
+	[SerializeField] void PlayerWalkSound()
+	{
+		SoundManager.instance.Play3DSoundAtPosition ( footstepSounds[Random.Range(0, footstepSounds.Length)],
+		                                              transform.position - transform.up );
```

Comment:
Why are you doing `transform.position - transform.up`? What's gained by the extra offset?